lib/automate_liveness_agent/liveness_update_sender.rb

```diff
@@ -55,15 +43,20 @@ def main_loop
     end
 
     def update
-      api_client.request(update_data)
+      api_client.request(update_payload)
     end
 
-    def update_data
-      name = "example_" + Time.now.to_i.to_s
-      sprintf(DATA_TEMPLATE, name, name)
+    def update_payload
+      {
+        "chef_server_fqdn" => URI(config.chef_server_url).host,
+        "source" => "liveness_agent",
+        "message_version" => "0.0.1",
+        "message_type" => "node_ping",
+        "organization_name" => config.chef_server_url.match(%r{/+organizations/+([a-z0-9][a-z0-9_-]{0,254})})[1],
```

Comment:
Also, using file path functions (like `File.split`) on `URI(uri).path` might be a little cleaner